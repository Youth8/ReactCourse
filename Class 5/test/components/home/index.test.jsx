import "../../__mocks__/epc-ui-layout-mock";

jest.mock("react-router-dom", () => ({
  __esModule: true, // this property makes it work
  Redirect: props => {
    if (props.to === "/login") {
      return "no login";
    }

    return "";
  }
}));

import React from "react";
import render from "react-test-renderer";
import Home from "../../../src/components/home";

test("test for home page", () => {
  const result = render.create(<Home />).toJSON();
  expect(result).toMatchSnapshot();
});
