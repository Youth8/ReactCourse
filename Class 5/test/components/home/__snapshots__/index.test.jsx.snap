// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for home page 1`] = `"no login"`;
